src/commands/run.rs

```diff
@@ -144,13 +144,28 @@ pub async fn command(args: Args, _json: bool) -> Result<()> {
         // this is for `rails c` and similar REPLs
     })?;
 
-    let exit_status =
-        tokio::process::Command::new(args.args.first().context("No command provided")?)
-            .args(args.args[1..].iter())
-            .envs(variables)
-            .status()
-            .await
-            .context("Failed to spawn command")?;
+    let mut args = args.args.iter().map(|s| s.as_str()).collect::<Vec<_>>();
+    if args.is_empty() {
+        bail!("No command provided");
+    }
+
+    // So the linter doesn't get mad. I think this is pretty ugly.
+    let child_process_name = match std::env::consts::OS {
+        "windows" => {
+            args.insert(0, "/C");
+            "cmd"
+        }
+        _ => {
+            args.remove(0);
```

Comment:
This doesn't seem to work

![image](https://user-images.githubusercontent.com/3044853/234071408-b26622c2-dc32-4c59-99b5-b89bacfb9d91.png)